scss/main.scss

```diff
@@ -498,7 +500,9 @@ h1,h2,h3 {
   justify-content: space-between;
   text-align: center;
   font-weight: bold;
-  margin: 0 auto 4.39rem;
+  margin: auto;
+  margin-top: 1.5rem;
+  margin-bottom: 1.5rem;
```

Comment:
This makes the spacing between the items incorrect. Can you please adjust?

**PR:**
<img width="1278" alt="Screen Shot 2021-02-10 at 9 21 33 PM" src="https://user-images.githubusercontent.com/9326234/107599470-3d4c6900-6be6-11eb-96ab-132f50e3545c.png">


**Design:**
<img width="555" alt="Screen Shot 2021-02-10 at 9 23 16 PM" src="https://user-images.githubusercontent.com/9326234/107599477-42a9b380-6be6-11eb-953a-aa2501fc522b.png">

Reply:
I see what you mean. I changed the adjusted the margins for the div containing the individual icons and labels instead of the whole row.

Reply:
Getting closer. The gap between the icons in this PR is ~50px now. Figma has it at ~70px.

Reply:
I'm a little confused. I thought the space between icons was supposed to be 48px. Let me know what I'm missing here. Thanks for your patience. Also, what is the tool that you are using that looks like a ruler? Is that a feature in Figma?
![Screen Shot 2021-02-21 at 7 08 31 PM](https://user-images.githubusercontent.com/64870162/108645548-d5651080-7478-11eb-91b1-1c0b47e5bace.png)

Reply:
Ah, my bad! I was looking at another part of the design...which Joseph has since labeled as "Don't use this!" 
This jives with the corrected design. Thanks!